cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/storage/StorageUpdateSnapshotCommand.java

```diff
@@ -0,0 +1,229 @@
+/**
+ * Copyright Pravega Authors.
+ * <p>
```

Comment:
Please, copy/paste the exact header as in other files and above adding these `<p>` markers and other variations w.r.t the official header.

Reply:
used one from other files. Doesnt seem to have any <p> markers.